### ISSUE TITLE
Regression: NRF52832 Access Port doesn't work anymore

### ISSUE THREAD

**Issue by maz3max** [blacksphere/blackmagic#522]
I am trying to flash a NRF52832 with activated APPROTECT.
That works with BMP Version v1.6.1-311-gfbf1963, but not with the current version (g3a6947a).
With the current version, the BMP crashes.

**Comment by maz3max** [comment]
I got following output after attaching to the debug port:
```
warning: while parsing target description: no element found
warning: Could not load XML target description; ignoring
Ignoring packet error, continuing...
Reply contains invalid hex digit 116
```
After that, the BMP does not respond anymore.

**Comment by maz3max** [comment]
I did some disection:
`19a8078f9934c99aaf1b88af4d41536d14c4d4b0 is the first bad commit`

**Comment by maz3max** [comment]
correction: I did some bisecting and this is the result:
`7bcf7f492443d077e35d4f39dcf2c04e7594befb is the first bad commit`

**Comment by maz3max** [comment]
when removing the support for "P" and "p" packets, it works again.

**Comment by UweBonnes** [comment]
i will have a look next week

**Comment by UweBonnes** [comment]
Fixed by #524
